webapp/src/resources.js

```diff
@@ -24,6 +24,11 @@ export const API_URL =
   process.env.VUE_APP_API_URL != null ? process.env.VUE_APP_API_URL : "http://localhost:5001";
 export const API_TOKEN = process.env.VUE_APP_API_TOKEN;
 
+export const QR_CODE_RESOLVER_URL =
+  process.env.VUE_APP_QR_CODE_RESOLVER_URL != null
+    ? process.env.VUE_APP_QR_CODE_RESOLVER_URL
+    : "https://purl.datalab-org.io";
```

Comment:
I think this is the bit I'm still unsure about; think we need another option for whether the link resolver should be turned on; or even a server value for "is this server in the federation", and if not, just use the datalab's URL as the resolver